uk.co.compucorp.civicrm.tasksassignments/js/test/services/document_test.js

```diff
@@ -81,12 +81,12 @@ define([
     }));
 
     beforeEach(function () {
-      spyOn(ContactService, 'get').and.callFake(fakePromise(contactMock.contactList));
+      spyOn(ContactService, 'get').and.callFake(fakePromise(contactMock.list()));
       spyOn(AssignmentService, 'get').and.callFake(fakePromise(assignmentMock.assignmentList));
       spyOn(ContactService, 'updateCache').and.returnValue(deferred.promise);
       spyOn(AssignmentService, 'updateCache').and.returnValue(deferred.promise);
 
-      $httpBackend.whenGET(/action=get&entity=contact/).respond({});
+      $httpBackend.whenGET(/action=get&debug=true&entity=contact/).respond({});
```

Comment:
It can not be temporary, it is always attached to url indicating whether to get the data in debug mode or  not.